app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,216 @@
+<!--
+  ~ Ichaival - Android client for LANraragi https://github.com/Utazukin/Ichaival/
+  ~ Copyright (C) 2022 Utazukin
+  ~
+  ~  This program is free software: you can redistribute it and/or modify
+  ~  it under the terms of the GNU General Public License as published by
+  ~  the Free Software Foundation, either version 3 of the License, or
+  ~  (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+  -->
+
+<resources>
+    <string name="app_name" translatable="false">Ichaival</string>
+    <string name="archive_thumb" translatable="false">thumb</string>
+    <string name="archive_label" translatable="false">name</string>
+
+    <string name="random">ランダム</string>
+    <string name="title_activity_settings">設定</string>
+
+    <!-- Example General settings -->
+    <string name="pref_header_general">全般</string>
+
+
+    <string name="server_address_preference" translatable="false">server_address</string>
+    <string name="server_location_name">サーバーアドレス</string>
+    <string name="clear_cache">キャッシュをクリアしています…</string>
+
+    <string name="load_previews">より多くの親指をロード</string>
+    <string name="bookmark">ブックマーク</string>
+    <string name="unbookmark">ブックマークを外す</string>
+    <string name="archive_details">詳細</string>
+    <string name="archive_gallery">親指</string>
+    <string name="read_button">読む</string>
+    <string name="tag_header">タグ</string>
+    <string name="bookmark_label">ブックマーク</string>
+    <string name="detail_menu_item">詳細</string>
+    <string name="api_key_pref" translatable="false">api_key</string>
+    <string name="api_key_pref_name">APIキー</string>
+    <string name="details_title">詳細</string>
+    <string name="thumbs_title">親指</string>
+    <string name="license_key" translatable="false">ライセンス</string>
```

Comment:
This shouldn't be translated since it is used as the key to access the setting, and isn't displayed to the user.